search/geocoder.cpp

```diff
@@ -745,7 +706,7 @@ void Geocoder::FillLocalitiesTable(BaseContext const & ctx)
 
 #if defined(DEBUG)
         ft.GetName(StringUtf8Multilang::kDefaultCode, city.m_defaultName);
-        LOG(LDEBUG, ("City =", city.m_defaultName, radius));
+        LOG(LINFO, ("City =", city.m_defaultName, "radius =", radius, "tfidf =", l.m_tfidf));
```

Comment:
Why? This code is compiled only when if defined(DEBUG), so there's no reason in LDEBUG.